src/components/Navbar/index.css

```diff
@@ -100,6 +100,10 @@
   padding-left: 20px;
   margin-top: -2px;
 }
+.search-Contributor{
```

Comment:
this is not related to the actual purpose, as we are searching users here instead of contributors. This class is suitable for contributors page